app/controllers/application_controller/ci_processing.rb

```diff
@@ -548,7 +548,7 @@ def evacuatevms
     recs = find_checked_items
     recs = [params[:id].to_i] if recs.blank?
     @record = find_by_id_filtered(VmOrTemplate, recs.first)
-    if @record.is_available?(:evacuate) && !@record.ext_management_system.nil?
+    if @record.supports_evacuate? && !@record.ext_management_system.nil?
```

Comment:
The has_active_ems? method contains one more condition other than ext_management_system.nil? which indicates evacuation operation might be supported even when ext_management_system.nil? is true.

`def has_active_ems?
    # If the VM does not have EMS connection see if it is using SmartProxy as the EMS
    return true unless ext_management_system.nil?
    **return true if host && host.acts_as_ems? && host.is_proxy_active?**
    false
  end`